qa/pull-tester/tests_config.ini.in

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2013-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# These environment variables are set by the build process and read by
+# rpc-tests.py
+
+[DEFAULT]
```

Comment:
Great! I wasn't aware of the get() method. Agree that removing the defaults section is much better.